src/utils/py/daos_api.py

```diff
@@ -1639,6 +1639,22 @@ def commit_tx(self, txn):
             raise DaosApiError("TX commit returned non-zero. RC: {0}"
                                .format(ret))
 
+    def close_tx(self, txn):
+        """ close out a transaction that is done being modified """
+
+        # container should be  in the open state
+        if self.coh == 0:
```

Comment:
It should of course be open but if we want to test a call where its not open should we let this through?

Reply:
Thanks for the comment. Script updated.